Console/Command/CakePHPUtilShell.php

```diff
@@ -0,0 +1,330 @@
+<?php
+App::uses('AppShellErrorHandler', 'Lib/Util');
+App::uses('Shell', 'Console');
+
+/**
+ * Application Shell
+ *
+ * Add your application-wide methods in the class below, your shells
+ * will inherit them.
+ *
+ * @package       app.Console.Command
+ *
+ * @property MailTask $Mail
+ */
+class CakePHPUtilShell extends Shell {
+
+	const SUCCESS_MESSAGE = 'Shell action ran successfully and without problems.';
+	/**
+	 * Tasks to use
+	 *
+	 * @var array
+	 */
+	public $tasks = array('Mail');
+
+	/**
+	 * Contains the debug value
+	 *
+	 * @var int
+	 */
+	protected $_debug;
+
+	/**
+	 * Contains all the exceptions made in the shell
+	 *
+	 * @var array
+	 */
+	protected $_exceptions = array();
+
+	/**
+	 * Set whether or not to mail shell result to an e-mail recipient.
+	 *
+	 * @var array
+	 */
+	protected $_mailTo = array(
+		'enabled' => 0, # 0 = Never send mail; 1 = Only mail on errors; 2 = Always mail, regardless of exit status
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use "//" instead of "#" to start this comment [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS2046)